opensource/project/software-req-win.md

```diff
@@ -10,7 +10,35 @@ This page explains how to get the software you need to build, test, and run the
 - GitHub account
 - Git
 
-## 1. Docker Windows containers
+## Prerequisites
+
+### 1. Windows Server 2016 or Windows 10 with all Windows updates applied.
+
+The major build number must be at least 14393. This can be confirmed, for example, by running the following from an elevated PowerShell prompt - this sample output is from a fully up to date machine as at mid-November 2016:
+
+
+    PS C:\> $(gin).WindowsBuildLabEx
+    14393.447.amd64fre.rs1_release_inmarket.161102-0100
+
+### 2. Git for Windows (or another git client) must be installed. 
+
+https://git-scm.com/download/win.
+
+### 3. The machine must be configured to run containers.
+
+For example, by followingthe quick start guidance at https://msdn.microsoft.com/en-us/virtualization/windowscontainers/quick_start/quick_start or https://github.com/docker/labs/blob/master/windows/windows-containers/Setup.md
+
+### 4. If building in a Hyper-V VM
+
+For Windows Server 2016 using Windows Server containers as the default option, it is recommended you have at least 1GB of memory assigned; For Windows 10 where Hyper-V Containers are employed, you should have at least 4GB of memory assigned. Note also, to run Hyper-V containers in a VM, it is necessary to configure the VM for nested virtualization.
+
+## Usage
+
+The following steps should be run from an (elevated*) Windows PowerShell prompt. 
```

Comment:
You either have an extra `*` after `elevated` or you are missing one before it.

Reply:
This isn't markdown but a reference to the following note.  Do you have a suggestion for making that more clear? A different symbol maybe?